.github/workflows/comment_on_first_contribution.yml

```diff
@@ -0,0 +1,54 @@
+name: Comment on first contribution
+
+on:
+  workflow_call:
+    inputs:
+      pull_request_message:
+        description: >
+          The Markdown message to comment when a contributor creates their first
+          pull request.
+        type: string
+        required: false
+      issue_message:
+        description: >
+          The Markdown message to comment when a contributor creates their first
+          issue.
+        type: string
+        required: false
```

Comment:
Can we provide a default message for both? Would avoid every repo to define its own custom message. Maybe even going one step further do we want each repo to customise this really? If so how do you expect them to customise this?